docs/reference/ilm/overview-ilm.asciidoc

```diff
@@ -0,0 +1,58 @@
+[role="xpack"]
+[testenv="basic"]
+[[overview-index-lifecycle-management]]
+== Index lifecycle management overview
+
+++++
+<titleabbrev>Overview</titleabbrev>
+++++
+
+You can create and apply {ilm-cap} ({ilm-init}) policies to automatically manage your indices 
+according to your performance, resiliency, and retention requirements. 
+Index lifecycle policies can trigger actions such as rollover, shrink, freeze, and delete. 
+
+Typically, you associate a lifecycle policy with an index template so it is applied   
+whenever an index is created from the template. 
```

Comment:
I thought the way it worked was:

* You define a match pattern when you create an index template.
* When you create an index that matches the match pattern, the index template is applied.

This sort of implies you create indices from the template space, which I don't think is the case.